test/riemann/codec_test.clj

```diff
@@ -118,3 +118,14 @@
         expected (dissoc original :empty-attr)]
     (is (= (msg {:events [expected]})
            (roundtrip {:events [original]})))))
+
+(deftest false-custom-attrbutes-should-not-be-removed-during-serialization
```

Comment:
Small suggestion here - unit test names are not a great place to encode the assertions you're making inside the test; that's what the optional second argument to `is` is used for. As a practical consideration, suppose you wanted to do a `lein test :only ...` to target this specific test case. You'd have to type out 66 characters here, instead of something shorter like `false-custom-attributes`.

Edit: I scrolled up and saw that the previous test is also named in this fashion (but not every test in this ns). 🤦‍♂ Feel free to leave this or change it, but wanted to note the idiom.